moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -1185,66 +1204,132 @@
+        </spacer>
+       </item>
+       <item row="1" column="1">
+        <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
-          <string>Current Scene Objects</string>
+          <string>Object Status</string>
          </property>
-         <layout class="QGridLayout" name="gridLayout_8">
-          <item row="1" column="1">
-           <widget class="QPushButton" name="import_url_button">
+         <layout class="QVBoxLayout" name="verticalLayout_12">
+          <item>
+           <widget class="QLabel" name="object_status">
+            <property name="enabled">
+             <bool>true</bool>
+            </property>
+            <property name="frameShape">
+             <enum>QFrame::StyledPanel</enum>
+            </property>
+            <property name="frameShadow">
+             <enum>QFrame::Raised</enum>
+            </property>
             <property name="text">
-             <string>Import &amp;URL</string>
+             <string>Status</string>
+            </property>
+            <property name="textFormat">
+             <enum>Qt::AutoText</enum>
+            </property>
+            <property name="WordWrap" stdset="0">
+             <bool>true</bool>
             </property>
            </widget>
           </item>
-          <item row="3" column="1">
-           <widget class="QPushButton" name="clear_scene_button">
-            <property name="text">
-             <string>Clear</string>
+         </layout>
+        </widget>
+       </item>
+       <item row="3" column="1">
+        <widget class="QGroupBox" name="groupBox_4">
+         <property name="sizePolicy">
+          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+           <horstretch>0</horstretch>
+           <verstretch>0</verstretch>
+          </sizepolicy>
+         </property>
+         <property name="title">
+          <string>Primitive Shapes</string>
+         </property>
+         <layout class="QHBoxLayout" name="horizontalLayout_12">
+          <item>
+           <widget class="QComboBox" name="shapes_combo_box">
+            <item>
+             <property name="text">
+              <string>box</string>
+             </property>
+            </item>
+            <item>
+             <property name="text">
+              <string>sphere</string>
+             </property>
+            </item>
+            <item>
+             <property name="text">
+              <string>cone</string>
+             </property>
+            </item>
+            <item>
+             <property name="text">
+              <string>cylinder</string>
+             </property>
+            </item>
+           </widget>
+          </item>
+          <item>
+           <widget class="QDoubleSpinBox" name="shape_size_x_spin_box">
+            <property name="singleStep">
+             <double>0.100000000000000</double>
+            </property>
+            <property name="value">
+             <double>1.000000000000000</double>
```

Comment:
Would much prefer a default value of `0.1` and a step value of `0.01`. It will be a smoother experience and fit the dimensions of the demo/tutorial robot.

Reply:
I guess it's not really important, I just feel like the default will force users to zoom out when their robot fills most of the screen. `0.2` and `0.05` could be a good compromise.

![Screenshot from 2020-06-04 18-34-39](https://user-images.githubusercontent.com/4535737/83740626-12426a80-a692-11ea-8f6f-2fcae2e51b19.png)

![Screenshot from 2020-06-04 18-24-21](https://user-images.githubusercontent.com/4535737/83740233-82042580-a691-11ea-97ba-cc59e66f253f.png)

Reply:
@felixvd  I agree, I'll update those values